osgeo_importer/tests.py

```diff
@@ -505,10 +620,10 @@ def test_file_add_view(self):
         self.assertTrue(len(response.context['object_list']) == 1)
         upload = response.context['object_list'][0]
         self.assertEqual(upload.user.username, 'non_admin')
-        self.assertEqual(upload.file_type, 'GeoJSON')
+        # self.assertEqual(upload.file_type, 'GeoJSON')
```

Comment:
1. Why is this being disabled
2. If reason for #1 is good, why not just delete it